File: Beginner/Ada School (ADASCOOL)/ada.java
import java.util.*;
import java.lang.*;
import java.io.*;


class Codechef {
	public static void main (String[] args) throws java.lang.Exception {
		Scanner scan = new Scanner (System.in);
		if(scan.hasNext()){
		    int t = scan.nextInt();
		    for(int i=0;i<t;i++){
		        int N = scan.nextInt();
		        int M = scan.nextInt();
		        if(N%2==0||M%2==0)
		            System.out.println("YES");
		        else
		            System.out.println("NO");
		    }
        }
        scan.close();
	}
}

File: Beginner/Ada School (ADASCOOL)/ada.cpp
#include <iostream>
using namespace std;

int main() {
	int T;
	cin >> T;
	while(T--){
	    int N,M;
	    cin >> N >> M;
	    if((N*M)%2==0){
	        std::cout << "YES" << std::endl;
	    }
	    else {
	        std::cout << "NO" << std::endl;
	    }
	}
	return 0;
}
